contracts/smart-contract-wallet/base/ModuleManager.sol

```diff
@@ -150,17 +150,30 @@ abstract contract ModuleManager is
         address to,
         uint256 value,
         bytes memory data,
-        Enum.Operation operation
+        Enum.Operation operation,
+        uint256 txGas
     ) public virtual returns (bool success) {
         // Only whitelisted modules are allowed.
         if (msg.sender == SENTINEL_MODULES || modules[msg.sender] == address(0))
             revert ModuleNotEnabled(msg.sender);
         // Execute transaction without further confirmations.
-        success = execute(to, value, data, operation, gasleft());
-        if (success) {
```

Comment:
we won't be doing this success check and emitting events anymore?

Reply:
yep. success is used as return value now. I don't see the need to emit `ExecutionFromModuleSuccess` or `ExecutionFromModuleFailure` since we have `ExecutionSucces/ExecutionFailure` in `Executor.execute`.
And this `ModuleTransaction` event consumes 10k+ gas, I was thinking how can we use it on backend and actually didn't find any reason that is worth 10k gas. 
In any case, we can easily add them back if at some point we need them